config/application.rb

```diff
@@ -44,5 +44,13 @@ class Application < Rails::Application
     # Fixes broken tailwind.css build for CI
     # @see https://github.com/rails/tailwindcss-rails/issues/153#issuecomment-1225895063
     config.assets.css_compressor = nil
+
+    # Lab test date range settings
+    config.x.lab_test_date_range = {
```

Comment:
BTW, there is a great library to manipulate configs
https://github.com/rubyconfig/config

JFYI.